src/app/components/Billboard/BillboardCurationGrid/index.tsx

```diff
@@ -0,0 +1,46 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import styles from './index.styles';
+import CurationPromo from '../../Curation/CurationPromo';
+import { CurationGridProps } from '../../Curation/types';
+
+const BillboardCurationGrid = ({ summaries }: CurationGridProps) => {
+  const calculatedSummaries = summaries.slice(0, 4);
```

Comment:
I guess we just keep in mind it'll always be the first 4 so if more are added in Tipo, editorial will need to manage the numbers to avoid incorrect promos appearing.

Is it worth renaming this something like `topBillboardSummaries` or `billboardSummaries`?